index.css

```diff
@@ -2,6 +2,11 @@
 margin: 0;
 padding: 0;
 }*/
+@import url('https://fonts.googleapis.com/css?family=Oxygen&display=swap');
```

Comment:
IMO I would put the import at the top of the file but I won't let this block the release.